src/main/java/com/minecolonies/core/network/messages/server/colony/building/OpenCraftingGUIMessage.java

```diff
@@ -29,7 +29,7 @@ public class OpenCraftingGUIMessage extends AbstractBuildingServerMessage<IBuild
     /**
      * The type of container.
      */
-    private int id;
+    protected int id;
```

Comment:
Is this change strictly needed?
You can just create a custom message class and use that instead of modifying and extending this one.